contrib/internal/httptrace/httptrace.go

```diff
@@ -91,7 +95,6 @@ func ippref(s string) *netaddr.IPPrefix {
 }
 
 // genClientIPSpanTags generates the client IP related tags that need to be added to the span.
-// See https://datadoghq.atlassian.net/wiki/spaces/APS/pages/2118779066/Client+IP+addresses+resolution
```

Comment:
Public doc for this: https://docs.datadoghq.com/tracing/configure_data_security#configuring-a-client-ip-header